Makefile

```diff
@@ -6,7 +6,7 @@ FLAGS := -v
 
 .PHONY: all build clean install-deps lint tidy verify $(PROTODIR) $(WPDIR)
 
-all: build test node
+all: build test node gitaly
```

Comment:
@g4s8 should we also change `node` and `gitaly` to `degitx` and `degitx-gitaly`?

Reply:
@stepanvalyavskiy you mean `Makefile` targets? I think yes